src/app/checkout/step-3/step-3.tpl.html

```diff
@@ -159,7 +159,7 @@
       <div class="panel-body">
         <div class="row">
           <div class="col-md-12">
-            {{$ctrl.premiumName}}
+            {{$ctrl.premiumSelectLabel}}
```

Comment:
Seems like you could have put the single quotes here instead of making a new variable.

Reply:
Though, I suppose it's in 2 places. So, either way is probably ok.

Reply:
I wasn't positive that I could use that string concatenation technique inside the html file, but if I can do that, then I would prefer to not create a new variable.

Reply:
You don't need string concatenation, it's just HTML so you can put the quotes outside of the `{{}}`